### PR TITLE
fix: Day of Week Language in Confirmation Emails

### LINKED ISSUE #17192: [CAL-4743] Day of week on confirmations
Found a bug? Please fill out the sections below. 👍

### Issue Summary

When clients receive a confirmation email, the day of the week is in Spanish, ie instead of Sat. for Saturday, it reads sab for Sabado.

### Steps to Reproduce

1. When a client books an appointment
2. The confirmation email includes the wrong language for the day of the week (everything else on the confirmation page is in English)

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

I expect it to use the dominant language (English) of the associated time zone (America/Los Angeles PST)

- What's happening right now that is different from what is expected

clients are seeing the confirmed email with Sab instead of Sat.

- This is an ideal result that the system should get after the tests are performed

Days of the week in English for America/Los Angeles timezone

- Browser version, screen recording, console logs, network requests: You can make a recording with [Bird Eats Bug](https://birdeatsbug.com/).
- Node.js version
- Anything else that you think could be an issue.

- How was this tested? This is quite mandatory in terms of bugs. Providing evidence of your testing with screenshots or/and videos is an amazing way to prove the bug and a troubleshooting chance to find the solution.
cal photo

<sub>[CAL-4743](https://linear.app/calcom/issue/CAL-4743/day-of-week-on-confirmations)</sub>

### PR DESCRIPTION
**Pull request by khushalmajoka**
## What does this PR do?

- Fixes #17192
- Ensures that the day of the week in confirmation emails is displayed in English for the specified time zone (America/Los Angeles PST) instead of in Spanish.
- This PR addresses the issue where the day of the week was shown in Spanish (e.g., 'Sab' for Saturday) instead of English ('Sat').

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

1. **Test Environment**: Ensure the test environment is set to the America/Los Angeles time zone.
2. **Booking Appointment**: Have a client book an appointment.
3. **Check Confirmation Email**: Verify that the confirmation email now displays the day of the week in English (e.g., "Sat" for Saturday).
4. **Test Cases**:
   - Confirm if the day name is consistent with the English format in multiple time zones.
   - Ensure that no other content in the confirmation email is affected.

## Checklist

- [x] I have read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- [x] My code follows the style guidelines of this project.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that my changes generate no new warnings.

### CODE REVIEW

**Review by Praashh — DISMISSED** [review]
this is also useful, thank you @khushalmajoka

**Comment by khushalmajoka** [comment]
Thanks a lot @Praashh! 🙏 I’ve always wanted to contribute to Cal.com, and now my first PR is about to be merged! 🎉 Even though it was a minor change, the experience was super rewarding and packed with learning. Looking forward to more contributions ahead! 🚀

File: packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { APP_NAME } from "@calcom/lib/constants";
 import { TimeFormat } from "@calcom/lib/timeFormat";
 import { WorkflowActions } from "@calcom/prisma/enums";
 
+require("dayjs/locale/en");
+
```
Comment (anikdhabal) [inline packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts:7]: I don't think it's a right way to fix that

**Review by anikdhabal — CHANGES_REQUESTED** [review]

File: packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts (hunk 1 above)
Comment (khushalmajoka) [reply packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts:7]: Okay, i will find some way to get it more structured or if u can suggest something on this that will be really helpful.

Comment (khushalmajoka) [reply packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts]: Hi @anikdhabal, Corrected it, Can u please check?

**Comment by khushalmajoka** [comment]
Hi @anikdhabal , Can u please review it.

**Comment by khushalmajoka** [comment]
Hi @anikdhabal , can u please review it.

**Comment by khushalmajoka** [comment]
Hi @keithwillcode , Is there anything needed from my end for this PR to be merged?

**Comment by anikdhabal** [comment]
@khushalmajoka Could you reproduce this issue on your end? I am unable to.

**Comment by khushalmajoka** [comment]
> @khushalmajoka Could you reproduce this issue on your end? I am unable to.

Hey @anikdhabal, what do u mean when u say reproduce the issue at my end?

**Comment by Praashh** [comment]
> > @khushalmajoka Could you reproduce this issue on your end? I am unable to.
> 
> Hey @anikdhabal, what do u mean when u say reproduce the issue at my end?

means, check the issue in you system and tell the steps to follow in order to test it.

**Comment by khushalmajoka** [comment]
ok got it may be u want to see this?

image

**Comment by khushalmajoka** [comment]
> > > @khushalmajoka Could you reproduce this issue on your end? I am unable to.
> > 
> > 
> > Hey @anikdhabal, what do u mean when u say reproduce the issue at my end?
> 
> means, check the issue in you system and tell the steps to follow in order to test it.

ohh okay okay I have previously tested it, let me see how I reproduced it earlier

**Comment by Praashh** [comment]
> > > @khushalmajoka Could you reproduce this issue on your end? I am unable to.
> > 
> > 
> > Hey @anikdhabal, what do u mean when u say reproduce the issue at my end?
> 
> means, check the issue in your system and tell the steps to follow in order to test it.

**Comment by khushalmajoka** [comment]
No Guyss, dont remember how i did it but It was the  issue in the reminder emails that come before the event, but it was the issue when I changed the timezone to America/LosAngeles in the settings.

**Comment by anikdhabal** [comment]
Can't reproduce the error

**Comment by nikwen** [comment]
I'm still seeing this issue. Here's what I did to reproduce it: https://github.com/calcom/cal.com/issues/17192#issuecomment-2486194561

Would it be possible to reopen this PR?

**Comment by nikwen** [comment]
@anikdhabal Would it be possible to reopen this PR?

The issue still occurs for me. See https://github.com/calcom/cal.com/issues/17192#issuecomment-2486194561

**Comment by anikdhabal** [comment]
@nikwen yeah will make a solution. Thanks for your patience🙏

**Comment by nikwen** [comment]
Thank you, @anikdhabal! 🙌

**Pull request closed without merging**